README.md

```diff
@@ -123,23 +123,24 @@ properties of the Entity object detailed in the following table (as per `sensor.
 
 | Name | Type | Default | Description |
 |------|:----:|:-------:|-------------|
-| entity ***(required)*** | string |  | Entity id of the sensor.
-| attribute | string | | Retrieves an attribute or [sub-attribute (attr1.attr2...)](#accessing-attributes-in-complex-structures) instead of the state
-| name | string |  | Set a custom display name, defaults to entity's friendly_name.
-| color | string |  | Set a custom color, overrides all other color options including thresholds.
-| unit | string |  | Set a custom unit of measurement, overrides `unit` set in base config.
-| aggregate_func | string |  | Override for aggregate function used to calculate point on the graph, `avg`, `median`, `min`, `max`, `first`, `last`, `sum`.
-| show_state | boolean |  | Display the current state.
-| show_indicator | boolean |  | Display a color indicator next to the state, (only when more than two states are visible).
-| show_graph | boolean |  | Set to false to completely hide the entity in the graph.
-| show_line | boolean |  | Set to false to hide the line.
-| show_fill | boolean |  | Set to false to hide the fill.
-| show_points | boolean |  | Set to false to hide the points.
-| show_legend | boolean |  | Set to false to turn hide from the legend.
-| state_adaptive_color | boolean |  | Make the color of the state adapt to the entity color.
-| y_axis | string |  | If 'secondary', displays using the secondary y-axis on the right.
-| fixed_value | boolean |  | Set to true to graph the entity's current state as a fixed value instead of graphing its state history.
-| smoothing | boolean |  | Override for a flag indicating whether to make graph line smooth.
+| entity ***(required)*** | string |         | Entity id of the sensor.
+| attribute | string |         | Retrieves an attribute or [sub-attribute (attr1.attr2...)](#accessing-attributes-in-complex-structures) instead of the state
+| name | string |         | Set a custom display name, defaults to entity's friendly_name.
+| color | string |         | Set a custom color, overrides all other color options including thresholds.
+| unit | string |         | Set a custom unit of measurement, overrides `unit` set in base config.
+| aggregate_func | string |         | Override for aggregate function used to calculate point on the graph, `avg`, `median`, `min`, `max`, `first`, `last`, `sum`.
+| show_state | boolean |         | Display the current state.
+| show_legend_state | boolean |  false  | Display the current state with the legend.
```

Comment:
This is the only required change, why other lines were changed? I see only whitespaces were added but it makes a "compare" view too crowded.

Reply:
When adding the false tag, my ide thought that it would look cleaner to have all blank spaces aligned.

Reply:
Well, not critical. Main thing that the presentation should not be changed. Can't test myself, was it changed?

Reply:
Visually nothing, it's only how the file look when not rendered that has been changed

Reply:
Suggest to change
"Display the current state with the legend."
to
"Display the current state as a part of the legend."